lib/ansible/module_utils/known_hosts.py

```diff
@@ -190,7 +190,10 @@ def add_host_key(module, fqdn, key_type="rsa", create_dir=False, port=SSH_KEYSCA
     elif not os.path.isdir(user_ssh_dir):
         module.fail_json(msg="%s is not a directory" % user_ssh_dir)
 
-    this_cmd = "%s -t %s %s" % (keyscan_cmd, key_type, fqdn)
```

Comment:
Yeah, I totally agree, I just didn't want to break "backwards-compatibility". What if right now Ansible is happily working on some system where `-p` option is not supported? For default port this can be omitted - so I omitted it.